src/js/components/create/where.js

```diff
@@ -43,7 +44,7 @@ export default class Where extends Component {
     const { data, name, addInput, removeInput } = this.props;
     const inputs = data.map((value, inputKey) => {
       return (
-        <View key={ Math.random() } style={inlineStyle.inputContainer}>
+        <View key={ inputKey } style={inlineStyle.inputContainer}>
```

Comment:
@minaorangina I had to replace Math.random() with inputKey from the array because the value for the input wasn't display anymore for some reason.